extensions/amp-jwplayer/0.1/test/test-amp-jwplayer.js

```diff
@@ -37,6 +37,12 @@ describes.realWin('amp-jwplayer', {
     jw.setAttribute('width', '320');
     jw.setAttribute('height', '180');
     jw.setAttribute('layout', 'responsive');
+    const html = htmlFor(env.win.document);
```

Comment:
@alanorozco  I've added this method but still am failing the unit test `renders with a playlist and parses contextual parameter`, if you have any suggestions as to how to resolve, that would be ideal. I have manually tested this and the functionality is working as expected.

Reply:
Since your query selection is `meta[property="og:title"]`, you need to change the selector in `withArgs` to match. Try that out and let me know how it goes :)

Reply:
Thanks, this worked! Appreciate the assist.